mycity/mycity/utilities/gis_utils.py

```diff
@@ -110,3 +113,14 @@ def _get_dest_addresses_from_features(feature_address_index, features):
             dest_address += " Boston, MA"
             dest_addresses.append(dest_address)
     return dest_addresses
+
+
+def geocode_address(m_address):
```

Comment:
What does the `m_` prefix stand for here?

Reply:
_My address? I have no idea, really. I borrowed this code from another pull request but based on tonight's discussion this function will be replaced by one that calls the rest API directly.

Reply:
sorry to jump in, but `m_` means member variable. It's commonly used in C++ (and Java I guess)